ui.frontend/src/constants.js

```diff
@@ -85,9 +85,25 @@ export const Constants = {
      */
     ARIA_INVALID : "aria-invalid",
 
+    /**
+     * aria attribute to mark the dragged component checked.
+     */
+    ARIA_CHECKED : "aria-checked",
+
     /**
      * Event to trigger when GuideBridge Initialisation Begins
      */
-    GUIDE_BRIDGE_INITIALIZE_START: "bridgeInitializeStart"
+    GUIDE_BRIDGE_INITIALIZE_START: "bridgeInitializeStart",
+
+    /**
+     * attribute to mark the dragged component disabled.
+     */
+    DISABLED : "disabled",
+
+    /**
+     * attribute to mark the dragged component checked.
+     */
+    CHECKED : "checked"
```

Comment:
I am not in favour of defining all the constants in one big value. We should put them in separate objects
HTML_ATTRS
FORM_DATA_ATTRS
ARIA_ATTRS